content/es/docs/concepts/storage/volumes.md

```diff
@@ -1198,7 +1198,7 @@ For more details, see the [FlexVolume](https://github.com/kubernetes/community/b
 
 La propagación del montaje permite compartir volúmenes montados por un contenedor para otros contenedores en el mismo Pod, o aun para otros pods en el mismo nodo.
 
-La propagación del montaje de un volumen es controlada por el campo `mountPropagation` en `Container.volumeMounts`. Sus valores son:
+La propagación del montaje de un volumen es controlada por el campo `mountPropagation` en `Container[*].volumeMounts`. Sus valores son:
```

Comment:
Minor update is required in the proposed changes here to ensure alignment with the 'en' PR https://github.com/kubernetes/website/pull/45337/files

```suggestion
La propagación del montaje de un volumen es controlada por el campo `mountPropagation` en `container[*].volumeMounts`. Sus valores son:
```

Reply:
Done, thanks